inquire/src/prompts/multiselect/mod.rs

```diff
@@ -172,6 +193,10 @@ where
     /// Default starting cursor index.
     pub const DEFAULT_STARTING_CURSOR: usize = 0;
 
+    /// Default cursor behaviour on filter input change.
+    /// Defaults to true if 'fuzzy' is enabled.
+    pub const DEFAULT_RESET_CURSOR: bool = cfg!(feature = "fuzzy");
```

Comment:
Honestly, I kind of prefer what you did here, and I'm ok with resetting cursor being the default behaviour (and not being configurable). It feels like a much better default

Reply:
The goal with that config flag was to change the behaviour as little as possible for people already using the library.
But it was necessary for the fuzzy search to feel good.
But it is now `true` per default and can be overwritten instead.